### ISSUE TITLE
[vagrant 1.2.3]Provisioning Multiple VM's in Parallel failed

### ISSUE THREAD

**Issue by lmayorga1980** [hashicorp/vagrant#1932]
Hi,

I tried to start multiple vagrant machines but it seems that i need to wait until one is complete to proceed with the next vm.
#### Environment

---
- OS: Mac OSX 10.8.4
- Vagrant 1.2.3
- Virtual Box 4.2.12
- Ruby 2.0.0p0
- Processor : Intel Core i7 - 64 bit
### Details

---

```
LT71707-MAC:oraculo lmo0$ vagrant up master
Bringing machine 'master' up with 'virtualbox' provider...
[master] Setting the name of the VM...
[master] Clearing any previously set forwarded ports...
[master] Fixed port collision for 22 => 2222. Now on port 2200.
[master] Creating shared folders metadata...
[master] Clearing any previously set network interfaces...
[master] Preparing network interfaces based on configuration...
[master] Forwarding ports...
[master] -- 22 => 2200 (adapter 1)
[master] Running any VM customizations...
[master] Booting VM...
[master] Waiting for VM to boot. This can take a few minutes.
The VM failed to remain in the "running" state while attempting to boot.
This is normally caused by a misconfiguration or host system incompatibilities.
Please open the VirtualBox GUI and attempt to boot the virtual machine
manually to get a more informative error message.


LT71707-MAC:oraculo lmo0$ vagrant destroy master
Are you sure you want to destroy the 'master' VM? [y/N] y
[master] Destroying VM and associated drives...
There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["unregistervm", "4b44abba-af83-4117-a563-4150490091a5", "--delete"]

Stderr: 0%...10%...20%...30%...
Progress state: VBOX_E_FILE_ERROR
VBoxManage: error: Machine delete failed
VBoxManage: error: Could not delete the medium storage unit '/Users/lmo0/VirtualBox VMs/oraculo_1373940514/box-disk1_1.vmdk'.
VBoxManage: error: VD: error VERR_FILE_NOT_FOUND opening image file '/Users/lmo0/VirtualBox VMs/oraculo_1373940514/box-disk1_1.vmdk' (VERR_FILE_NOT_FOUND)
VBoxManage: error: Details: code VBOX_E_FILE_ERROR (0x80bb0004), component Medium, interface IMedium
VBoxManage: error: Context: "int handleUnregisterVM(HandlerArg*)" at line 168 of file VBoxManageMisc.cpp
```

**Comment by semmerson** [comment]
@mitchellh I'm having the same problem.

Does VMware Player have the same problem (i.e., can concurrent `vagrant up` commands be executed if the provisioner is VMware)?

**Comment by michaeljsmalley** [comment]
@mitchellh How's it going? Is this something that has ever been implemented in any capacity? It's more of a nice-to-have than a must-have but just curious since the last post was on Mar 31, 2015 (almost two years ago).

**Comment by stephaneerard** [comment]
> 
> 
> VirtualBox doesn't like parallelism please don't do that! :) I'm going to add some features in the future to try to further limit this sort of thing from happening, but its difficult. :( It'd be better if VirtualBox behaved better under concurrent conditions.

Is this still true nowadays ? Thanks !
